app/assets/javascripts/global_listeners.js

```diff
@@ -1,144 +1,142 @@
-import Cookies from 'js-cookie';
-import ExecuteAttachinary from './libs/execute_attachinary';
-import KeycutView from './views/keycut_view';
+import Cookies from "js-cookie";
```

Comment:
NIT: looks like the only changes in this file were formatting changes, it would be good to revert them but I won't block the PR because of that.